Dshop/apps/payments/views.py

```diff
@@ -13,3 +15,68 @@ def stripe_config(request):
     if request.method == 'GET':
         stripe_config = {'publicKey': settings.STRIPE_PUBLISHABLE_KEY}
         return JsonResponse(stripe_config, safe=False)
+
+
+@csrf_exempt
```

Comment:
Why csrf_exempt?

Reply:
Because they handle requests that are not initiated by the user's browser but by Stripe or another external service. Disabling CSRF protection in these views is justified because it's not necessary in these cases, and enforcing CSRF protection could complicate the handling of these requests. 

I'm not very familiar with security issues yet, but research has shown that when connecting to Stripe, using csrf_extempt is justified.

What do you think?

Reply:
Thanks!

Reply:
Makes sense.